cypress/integration/13-contributionFlow.order.test.js

```diff
@@ -52,4 +52,28 @@ describe('Contribution Flow: Order', () => {
       cy.contains(`${user.firstName} ${user.lastName} is now a member of APEX's 'Sponsors' tier!`);
     });
   });
+
+  it('Can order with an existing orgnanization', () => {
+    cy.clock(Date.parse('2042/05/25'));
+
+    const visitParams = { onBeforeLoad: mockRecaptcha };
+    // Login and rediect to the order page
+    cy.login({ redirect: '/apex/contribute/tier/470-sponsors', visitParams });
+
+    // Select 'Test Collective' organization profile
+    cy.get('[type="radio"][value="10880"]').check();
```

Comment:
We should ideally not rely on organizations created in previous tests. Maybe the [`createCollective` cypress command ](https://github.com/opencollective/opencollective-frontend/blob/3d3ac6eda80377864563bf829263b7bcc0677503/cypress/support/commands.js#L45) can help to make this more flexible?

Reply:
This would lead to creating multiple organization on every cypress run, is that good?

Reply:
Yep creating multiple organizations is not a problem. Of course the test must still work when we run it multiple times.

Also relying on the id (`[value="10880"]`) is not generic (it will break if we create another collective in one of the previous tests) so we should select the entry based on the label - either by having a random organization name to ensure it's unique or by using `:first`/`:last` selectors.